extensions/sftp/tests/PutSFTPTests.cpp

```diff
@@ -504,7 +504,9 @@ TEST_CASE_METHOD(PutSFTPTestsFixture, "PutSFTP set mtime", "[PutSFTP]") {
   testController.runSession(plan, true);
 
   testFile("nifi_test/tstFile1.ext", "content 1");
-  testModificationTime("nifi_test/tstFile1.ext", 3000000000LL);
+  using namespace std::chrono;  // NOLINT(build/namespaces)
```

Comment:
nitpick: I would use `std::literals::chrono_literals` namespace here instead as we allow that in the linter and use the fully qualified namespaces for the type declaration.